src/frontends/pytorch/src/op/prod.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/reduce_prod.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_prod(const NodeContext& context) {
+    num_inputs_check(context, 2, 4);
```

Comment:
could you please add supported operation signatures as comment for better understanding if something changing in future pytorch versions?

Reply:
Okay, will add in next PR